### PR TITLE
Update Japanese materials

### PR DESCRIPTION
**Pull request by dzeyelid**
I updated Japanese docs along with current English docs.
Please review them.

I also some changes to English docs to improve them.

### CODE REVIEW

**Comment by ppremk** [comment]
@dzeyelid 👋🏽 are you able to paste the local test run screen capture and results in here please. This will help us to keep audit that this PR when merge works as expected and does not break the manual in general. 🙇🏽

**Comment by junkofujiwara** [comment]
verified the translation.

**Comment by ppremk** [comment]
Translation verified by @junkofujiwara. Good to merge when all test have passed.

**Comment by dzeyelid** [comment]
@junkofujiwara Thank you for verifying!
@ppremk I fixed all linter errors, including some that I did not change but were shown as errors by the linter. Please confirm it 🙏

File: docs/02a_activity_create_github_issue.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ Follow these steps to create an issue in the class repository:
 1. Type the following in the Subject line: `YOUR-USERNAME Workflow`
 1. In the body of the issue, include the text below:
 
-   ```md
-   YOUR-USERNAME will choose an image, add a caption, and add both to a file.
+```md
+YOUR-USERNAME will update a file by replacing a caption.
```
Comment (parkerbxyz) [inline docs/02a_activity_create_github_issue.md:11]:
```suggestion
   ```md
   YOUR-USERNAME will update a file by replacing a caption.
```

Please restore the spaces here so the code block will be part of the list. 🙂

File: docs/12b_resolving_merge_conflicts.md [hunk 2]
```diff
@@ -15,4 +15,67 @@ Let's try to create a merge conflict, and fix it together. You and a partner wil
...
+   git add _slides/##-USERNAME.md
+   git commit
+   ```
+
+   1. Push the feature branch up to the remote, and see the resolution in the pull request.
+
+   ```sh
+   git push
+   ```
+
```
Comment (parkerbxyz) [inline docs/12b_resolving_merge_conflicts.md:78]: Please indent each of the fenced code blocks an additional 3 spaces so they will be inline with the list items they correspond to. 

## Example

### Before

1. When you see the merge conflict in the _second_ pull request titled `USERNAME resolve conflict`, work by yourself to resolve the merge conflict.
   1. Working locally, switch to `main` branch and pull updates from remote.

   ```sh
   git switch main
   git pull
   ```

   1. Fetch the `USERNAME-modify-conflict` branch from remote and switch to the same branch locally.

   ```sh
   git fetch
   git switch USER-modify-conflict
   ```

### After

1. When you see the merge conflict in the _second_ pull request titled `USERNAME resolve conflict`, work by yourself to resolve the merge conflict.
   1. Working locally, switch to `main` branch and pull updates from remote.

      ```sh
      git switch main
      git pull
      ```

   1. Fetch the `USERNAME-modify-conflict` branch from remote and switch to the same branch locally.

      ```sh
      git fetch
      git switch USER-modify-conflict
      ```

<sup>Reference: https://www.markdownguide.org/basic-syntax/#adding-elements-in-lists</sup>

File: docs/01_getting_ready_for_class.md [hunk 3]
```diff
@@ -113,18 +113,24 @@ You can use almost any text editor, but we have the best success with the follow
...
 atom .
 ```
 
+Or the following command can also open the current directory with Visual Studio Code:
+
+```sh
+code .
+```
+
 _**Note:** If you are working on a Mac, you will need to Install Shell Commands from the Atom menu, this happens as part of the installation process for Windows._
```
Comment (parkerbxyz) [inline docs/01_getting_ready_for_class.md:134]: Let’s move this note up so it’s just below the `atom .` command.

File: docs/17_protected_branches.md [hunk 4]
```diff
@@ -29,7 +29,7 @@ Repository maintainers can define exactly which people and teams need to review
 Let's create a CODEOWNERS file:
 
 1. Go out to the **Code** tab of your repository.
-1. Click the **Create new file** button.
+1. Click the **Add file** drop down and Click the **Create new file**.
```
Comment (parkerbxyz) [inline docs/17_protected_branches.md:32]:
```suggestion
1. Click the **Add file** drop down and then **Create new file**.
```

File: docs/22_merge_strategies_rebase.md [hunk 5]
```diff
@@ -39,6 +39,7 @@ One of the most common uses of rebase is to eliminate recursive merges and creat
 
 #### Setup
 
+1. First, switch back to `main` branch: `git switch main`
```
Comment (parkerbxyz) [inline docs/22_merge_strategies_rebase.md:42]:
```suggestion
1. First, switch back to the `main` branch: `git switch main`
```

File: docs/ja/12b_resolving_merge_conflicts.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+   git add _slides/##-USERNAME.md
+   git commit
+   ```
+
+   1. そのフィーチャーブランチをリモートにプッシュします。
+
+   ```sh
+   git push
+   ```
+
```
Comment (parkerbxyz) [inline docs/ja/12b_resolving_merge_conflicts.md:78]: Let’s fix the indentation of the fenced code blocks here too.

File: docs/12b_resolving_merge_conflicts.md (hunk 2 above)
Comment (dzeyelid) [reply docs/12b_resolving_merge_conflicts.md:78]: Thanks for the advice!

**Comment by dzeyelid** [comment]
@parkerbxyz Thanks for your review! I applied your feedback. Please confirm it. 🙏

**Review by parkerbxyz — APPROVED** [review]
These changes look great. Thanks again, @dzeyelid! 💖

**Pull request merged**